CONTRIBUTING.md

```diff
@@ -1,10 +1,11 @@
 # Contributing to the Hyperledger Besu Documentation
-## :tada: Thanks for taking the time to contribute! :tada:
 
+## Thanks for taking the time to contribute!
```

Comment:
Maybe we should not make it a heading as the ! will break the linter.

Reply:
I'll remove the !